Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -419,8 +419,9 @@ define([
             instanceScale.x = 1.0;
             instanceScale.y = 1.0;
             instanceScale.z = 1.0;
-            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT);
+            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT, 1);
```

Comment:
Yeah I see the inconsistently there. As far as I can tell

```js
if (subArray.length === 1) {
    return subArray[0];
}
```

won't be reached by any of the code, as most areas would call `getGlobalProperty` instead. Perhaps that check should be removed, then the binary and json array sections would be consistent.